### ISSUE TITLE
[Feature Request] Multiple Colorschemes

### ISSUE THREAD

**Issue by XIlance** [Flowit-Game/Flowit#10]
It would be great if I could change the colorscheme. It's very difficult for me to distinguish red and green.

**Comment by ywnico** [comment]
I'd like to second this, as the game is not really playable when red and green are hard to distinguish.

As an example, I tried making a more accessible palette, based on https://jfly.uni-koeln.de/color/#pallet (with red darkened and gray added), with RGB values (191, 75, 0), (89, 187, 242), (0, 85, 133), (246, 202, 24), (155, 155, 155). These colors should be distinguishable with deuteranopia, protanopia, and tritanopia. Although I can't vouch for my design sense in choosing them.

![game_example_pair](https://user-images.githubusercontent.com/96934073/194372466-85fae1ff-c1f1-4bec-9fa9-948bafb2a4dd.png)

I'm not sure what the best approach to implementing an alternate color option would be. My understanding of the code is very limited, but it seems like it might be hard to reload a different texture.png while the app is running? If that's the case, maybe texture.png could contain both colorschemes in full, side by side, and changing the color preference would just add a pixel offset to the texture calls?

**Comment by ByteHamster** [comment]
I think it shouldn't be too hard loading a different texture.png at runtime. By the way, back then when I came up with the colors, I did test the color scheme with Android's deuteranopia, protanopia, and tritanopia simulation, and additionally tested it with a real person. The result was a compromise between aesthetics and differentiability. Anyway, I'm not opposed to merging a PR that adds more color schemes :)
